CONTRIBUTING.md

```diff
@@ -0,0 +1,155 @@
+# Contributing to Bench Routes Dashboard
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to `Bench-routes Dashboard`. These are mostly guidelines, not rules.
+Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Code of Conduct
+
+This project and everyone participates in it is governed by this [Code of Conduct](CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code. Please report unacceptable behavior to [**benchroutes@gmail.com**](mailto:benchroutes@gmail.com).
+
+## How Can I Contribute?
+
+Firstly, if you are new to the project, kindly go through [README.md](https://github.com/bench-routes/dashboard/blob/master/README.md) to setup the project
+in your local machine. Then have a quick look at [GUIDE.md]() to understand working of all the components present in the project. You can also have a look at our project [design documentation]() to have more insights on our project structure and design.
```

Comment:
Please provide the link here for GUIDE.md and design documentation

Reply:
Ohh missed that thanks